target/linux/ramips/image/mt7620.mk

```diff
@@ -631,6 +631,20 @@ define Device/netgear_ex3700
 endef
 TARGET_DEVICES += netgear_ex3700
 
+define Device/netgear_ex6130
+  MTK_SOC := mt7620a
+  NETGEAR_BOARD_ID := U12H319T50_NETGEAR
+  BLOCKSIZE := 4k
```

Comment:
Hmm, while this SPI flash supports 4k sectors, OpenWrt addresses SPI flash using 64k large blocks.

What this mismatch potentially does is break the jffs2 end marker, leading to incorrectly performed sysupgrades or non-working `firstboot`/`jffs2erase`.

Can you post the output of `cat /proc/mtd` to verify blocks are 4k large?

Reply:
Here you are:

```
root@EX6130:~# cat /proc/mtd 
dev:    size   erasesize  name
mtd0: 00030000 00001000 "u-boot"
mtd1: 00010000 00001000 "config"
mtd2: 00010000 00001000 "factory"
mtd3: 00790000 00001000 "firmware"
mtd4: 001b2582 00001000 "kernel"
mtd5: 005dda7e 00001000 "rootfs"
mtd6: 000f7000 00001000 "rootfs_data"
mtd7: 00010000 00001000 "board_data"
mtd8: 00010000 00001000 "nvram"
```

Reply:
Okay, this is the first time I'm seeing 4k large blocks. but all good in this case :+1: